app/src/main/res/layout/unanswered_survey_view.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView
+  style="@style/ActivityFeedCard"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:card_view="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:layout_marginBottom="@dimen/grid_new_2"
+  android:focusable="true"
+  card_view:cardCornerRadius="0dp"
+  card_view:cardUseCompatPadding="true">
+
+  <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
is this a duplicate `xmlns:android="http://schemas.android.com/apk/res/android"` ?